christfellowshipchurch/web-app-v2
# Podcast Bug Fix

### About
This PR fixes the podcast notes bug when first accessing the page it does not load the podcast, only the cover image and to 
listen to the podcast the page must be reloaded.

### Test Instructions
1. Go to [sisterhood-shownotes-plus](https://web-app-v2-git-podcast-bug-christ-fellowship-church.vercel.app//studies/sisterhood-shownotes-plus) on the testing link, it is recommended to use incognito mode to ensure there is no cache affecting results.
2. Click on one of the episodes at the bottom and the podcast should, not just the cover image.
3. Go to the previous page, and the image for that page should load, not the podcast for the episode clicked on earlier.

### Screenshots
<img width="1173" alt="Screenshot 2024-01-19 at 9 32 20 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/6530faf7-2f34-42a2-9c23-0ca81b804a00">


### Closes Tickets
[CFDP-2792](https://christfellowshipchurch.atlassian.net/browse/CFDP-2792)



[CFDP-2792]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2792?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
